runtime/vm/resolvefield.cpp

```diff
@@ -78,13 +77,19 @@ VMINLINE bool createClassLoaderHotFieldPool(J9JavaVM *javaVM, J9ClassLoader* cla
 VMINLINE void createClassHotFieldsInfo(J9JavaVM *javaVM, J9Class* clazz, uint8_t fieldOffset, int32_t reducedCpuUtil, uint32_t reducedFrequency);
 VMINLINE void addOrUpdateHotField(J9JavaVM *javaVM, J9Class* clazz, uint8_t fieldOffset, int32_t reducedCpuUtil, uint32_t reducedFrequency);
 
+J9ROMFieldShape*
```

Comment:
This seems unnecessary - why not just make the real thing non-static?

Reply:
There is another definition of findField in cthelper

Reply:
It's static and in another module, so I doubt it will cause any issue. If anything, I'd be tempted to rename that one rather than adding the wrapper here.